src/builder_state.rs

```diff
@@ -276,7 +292,7 @@ impl<TYPES: NodeType> BuilderProgress<TYPES> for BuilderState<TYPES> {
         }
         // Do the validation check
         else if da_msg.proposal.data.view_number.get_u64()
-            != self.built_from_view_vid_leaf.0.get_u64() + 1
+            != self.built_from_info.view_number.get_u64() + 1
         {
             tracing::info!("View number is not equal to built_from_view + 1, so returning");
```

Comment:
And maybe update this message to match.

Reply:
https://github.com/EspressoSystems/hotshot-builder-core/pull/77/commits/1b6f986c134cabf2e600325c4a4bb2361fce2a7f

Reply:
I think that message is good to debug. I had to explicitly kept it like that.